scheduler/mesosphere-dcos/framework-cleanup.md

```diff
@@ -83,7 +83,10 @@ do
         dcos node ssh --mesos-id=${ip} --master-proxy 'sudo rm /etc/systemd/system/portworx.service -f'
         dcos node ssh --mesos-id=${ip} --master-proxy 'sudo rm /etc/systemd/system/dcos.target.wants/portworx.service -f'
         dcos node ssh --mesos-id=${ip} --master-proxy 'sudo systemctl daemon-reload'
+	dcos node ssh --mesos-id=${ip} --master-proxy 'sudo chattr -i /etc/pwx/.private.json'
```

Comment:
some indentation issue here

Reply:
Fixed. The online editor automatically adds tabs